profiler/profiler_quickstart/busybench.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START profiler_quickstart]
+
+// Sample profiler_quickstart simulates a CPU-intensive workload for profiler.
+package main
+
+import (
+	"log"
+	"runtime"
+
+	"cloud.google.com/go/profiler"
+)
+
+func busyloop() {
+	for {
+		load()
+		runtime.Gosched()
+	}
+}
+
+func load() {
+	for i := 0; i < (1 << 20); i++ {
+	}
+}
+
+func main() {
+	runtime.GOMAXPROCS(8)
+	err := profiler.Start(
+		profiler.Config{
+			Service:              "hello-profiler",
+			NoHeapProfiling:      true,
+			NoGoroutineProfiling: true,
+			DebugLogging:         true,
+		})
```

Comment:
Just a nit, but if I were writing this code, I'd probably put the profiler.Config on the same line as the call to profiler.Start.